locales/core/de.json

```diff
@@ -0,0 +1,207 @@
+{
+  "account": "Account",
+  "addCard": "Kreditkarte hinzufügen",
+  "agreeToTermsLabel": "Ich stimme den Nutzungsbedingungen zu",
+  "album": "Album",
+  "allTime": "Aller Zeiten",
+  "areYouSureToDelete": "Alle Daten werden dauerhaft gelöscht. Bist du dir absolut sicher, dass du dein Konto löschen willst?",
+  "artist": "Künstler",
+  "backToLogin": "Zurück zum Login",
+  "basic": "Basic",
+  "billMonthly": "Wird monatlich abgerechnet",
```

Comment:
This is a little too long, can it be shortened?

Reply:
Changed it to "Monatliche Abrechnung". Short enough?

Reply:
Seems like this is long enough, we need to change some formatting on the `core` side to make it fit better. Unless there's a shorter way, I think we should leave it the way it is.